services/web-app/pages/assets/index/index.module.scss

```diff
@@ -34,6 +34,10 @@
   font-weight: 600;
 }
 
+.link {
+  @include type-style('body-long-02');
```

Comment:
Do we need the token here, could we not just do `font-size: inherit` so it picks up on the token from the paragraph and matches the size? That way I think we would be covered in the future if we wanted to add a link to any of the larger type sections.

Reply:
That works too. Updating.